### PR TITLE
AC-1088 - "Let's Code" item for Adding a sending domain

### PR DESCRIPTION
**Pull request by shrirupa**
AC-1088 - "Let's Code" item for Adding a sending domain

### What Changed
 - Action links to the sending domain
 - Button is outline gray after completing condition
 - Guide on first visit to Edit Sending Domain page highlighting the set up sending section (https://sparkpost.invisionapp.com/share/DHULLYL45GB#/screens/390961108) 
 - completion condition : verifying sending domain
- Added story for BreadCrumb

### How To Test
 - Enable flag - 'messaging onboarding'
 - Click on 'Add a Sending Domain' button in Let's Code Checklist.
 - Check if the BreadCrumb story is ok. (npm run storybook-dev)
 - Check if the pendo guide matches mocks and appears while editing domain for first time.
### To Do
- [ ] Address Feedback

### CODE REVIEW

File: src/components/breadcrumbs/BreadCrumbs.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
```
Comment (PatrickSison) [inline src/components/breadcrumbs/BreadCrumbs.stories.mdx]: 🎉

File: src/pages/dashboard/components/GettingStartedGuide.js [hunk 2]
```diff
@@ -80,6 +89,9 @@ export const GettingStartedGuide = ({ onboarding = {}, history, setAccountOption
         setOnboardingAccountOption({ invite_collaborator_completed: true });
         history.push('/account/users');
         break;
+      case 'Add Sending Domain':
```
Comment (PatrickSison) [inline src/pages/dashboard/components/GettingStartedGuide.js]: Should this get a test in `GettingStartedGuide.js`?

Comment (shrirupa) [reply src/pages/dashboard/components/GettingStartedGuide.js]: Definitely! I will add it. 👍

File: src/pages/dashboard/components/tests/GettingStartedGuide.test.js [hunk 3]
```diff
@@ -65,7 +65,7 @@ describe('GettingStartedGuide', () => {
     );
   });
 
-  it('should navigate to summary report when Exlplore Analytics button is clicked', () => {
+  it('should navigate to summary report when Explore Analytics button is clicked', () => {
```
Comment (PatrickSison) [inline src/pages/dashboard/components/tests/GettingStartedGuide.test.js]: Good catch

**Pull request merged**